doc/nrf/known_issues.rst

```diff
@@ -417,6 +417,13 @@ nRF5
 nRF5340
 =======
 
+.. rst-class:: v2-2-0
+
+NCSDK-20070: The :ref:`direct_test_mode` antenna switching functionality does not work on nRF5340 DK with nRF21540 EK shield.
+  The antenna select DTM command does not have any effect because the GPIO pin which controls antenna is not forwarded to nRF5340DK network core.
```

Comment:
```suggestion
NCSDK-20070: The :ref:`direct_test_mode` antenna switching does not work on the nRF5340 DK with the nRF21540 EK shield
  The antenna select DTM command does not have any effect because the GPIO pin which controls antenna is not forwarded to nRF5340DK network core.
```